node/src/GlideClusterClient.ts

```diff
@@ -1257,28 +1258,36 @@ export class GlideClusterClient extends BaseClient {
     /**
      * Returns the number of subscribers (exclusive of clients subscribed to patterns) for the specified shard channels.
      *
-     * Note that it is valid to call this command without channels. In this case, it will just return an empty map.
-     * The command is routed to all nodes, and aggregates the response to a single map of the channels and their number of subscriptions.
-     *
      * @see {@link https://valkey.io/commands/pubsub-shardnumsub/|valkey.io} for details.
+     * @remarks When in cluster mode, the command is routed to all nodes, and aggregates the response into a single list.
```

Comment:
This command exists on cluster client only
```suggestion
     * @remarks The command is routed to all nodes, and aggregates the response into a single list.
```